src/SignhostAPIClient/Rest/JsonConverters/LevelEnumConverter.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Reflection;
+using Newtonsoft.Json;
+using Signhost.APIClient.Rest.DataObjects;
+
+namespace Signhost.APIClient.Rest.JsonConverters
+{
+	/// <inheritdoc/>
```

Comment:
I am fine with the inheritdoc on all the inheritted properties. But some summary on what this converter is for might be welcome.